modules/ossm-threescale-cr.adoc

```diff
@@ -36,6 +36,11 @@ The adapter includes a tool that allows you to generate the `handler`, `instance
 |Yes
 |
 
+|`--backend-url`
+|3scale backend URL. If set, it overrides the value read from system configuration
```

Comment:
```suggestion
|3scale backend URL. If set, it overrides the value that is read from system configuration
```

Reply:
+1, I also think that in the main document, where we are describing the functionality of the handler, this should be discussed further as its important

Reply:
Done!